lib/preload.js

```diff
@@ -20,7 +20,7 @@ const { R_OK, W_OK } = fs.constants;
 
 const DOCKER_IMAGE_TAG = 'balena/balena-preload';
 const DISK_IMAGE_PATH_IN_DOCKER = '/img/balena.img';
-const SPLASH_IMAGE_PATH_IN_DOCKER = '/img/resin-logo.png';
+const SPLASH_IMAGE_PATH_IN_DOCKER = '/img/balena-logo.png';
```

Comment:
This name only applies to the Docker container created by balena-preload in order to execute the `preload.py` Python program. It is not the name of the splash image file in the preloaded balenaOS image.